.ci/ci_functions.sh

```diff
@@ -0,0 +1,46 @@
+function configure () {
+    rm -rf $NFS_WORKSPACE
+    rm -rf ./nccl-rdma-sharp-plugins
+    cd $NFS_WORKSPACE_ROOT
+    git clone git@github.com:Mellanox/nccl-rdma-sharp-plugins.git 
+
+    cp -r ./SRC/.ci ./nccl-rdma-sharp-plugins/
+    rm -rf ./nccl-rdma-sharp-plugins/.ci/cfg/*
+
+    mkdir -p ./nccl-rdma-sharp-plugins/.ci/cfg/$HOST1/sharp_conf
+    mkdir -p ./nccl-rdma-sharp-plugins/.ci/cfg/$HOST2/sharp_conf
+
+    printf "$HOST1\n$HOST2\n">./nccl-rdma-sharp-plugins/.ci/cfg/$HOST1/hostfile
+    printf "log_verbosity 5\n">./nccl-rdma-sharp-plugins/.ci/cfg/$HOST1/sharp_conf/sharp_am.cfg
```

Comment:
Do we really need log level 5? Maybe 3 is enough?

Reply:
set 3